### PR TITLE
Create 2021-07-14 posts

### PR DESCRIPTION
**Pull request by arianna-merritt**

### CODE REVIEW

**Comment by arianna-merritt** [comment]
Links for review:
**EN**: https://deploy-preview-45--blog-tbs.netlify.app/2021/07/14/effective-links
**FR**: https://deploy-preview-45--blogue-sct.netlify.app/2021/07/14/liens-plus-efficaces

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+date: 2021-07-14
+description: " "
+title: "3 étapes pour rendre les liens plus efficaces"
+---
+
+Sur le Web, les internautes sont à la recherche de réponses. La manière dont nous configurons une page influence la façon dont les internautes se déplacent dans notre contenu. Lorsque nous ajoutons des liens, nous pouvons fournir davantage de contexte ou amener les internautes à l'étape suivante d'un processus sans alourdir une page avec trop de renseignements. 
+
+Un lien bien placé peut fournir une «&nbsp;piste d'information&nbsp;» que les gens peuvent suivre facilement pour accomplir une tâche. Mais l'ajout d'un mauvais lien au mauvais endroit peut envoyer les gens dans des boucles frustrantes et les empêcher d'utiliser votre page Web avec succès.
+
+Lorsque vous concevez un contenu comprenant des liens, concentrez-vous sur les trois clés du succès :
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:12]:
```suggestion
Lorsque vous concevez un contenu comprenant des liens, concentrez-vous sur les trois clés du succès&nbsp;:
```
Colons could use non-breakable space too in French.

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+2. Où placer les liens
+3. Comment rédiger un bon texte de lien
+
+## Tous les liens sont des appâts à clics
+
+Ce n'est pas parce que nous pouvons ajouter des liens partout dans notre contenu que nous devrions le faire. 
+
+Les liens agissent comme des panneaux de signalisation pour les bretelles d'accès sur une autoroute. Ils guident les utilisateurs. Ils attirent également l'attention. Les gens supposent qu'un lien est là pour être suivi.
+
+Imaginez une autoroute à 8 voies très fréquentée avec des panneaux pointant dans toutes les directions. C'est déroutant, non ? Dois-je rester sur ma voie ou prendre la bretelle de sortie ?!
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:24]: What’s our stance on spaces before question/exclamation marks?

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+Si le lien ne soutient pas vraiment la tâche de la page, ne le mettez pas sur la page.
+
+## Placez les liens de façon à ce que les gens les trouvent
+
+L'emplacement des liens sur une page a un impact important sur leur efficacité.
+
+### Évitez l'éparpillement des liens
+
+Éparpiller les liens ici et là au milieu des phrases et des paragraphes comporte les inconvénients suivants :
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:38]:
```suggestion
Éparpiller les liens ici et là au milieu des phrases et des paragraphes comporte les inconvénients suivants&nbsp;: 

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+Éparpiller les liens ici et là au milieu des phrases et des paragraphes comporte les inconvénients suivants : 
+
+- Les liens sont faciles à manquer lors du balayage de la page. 
+- Le flux de la phrase est interrompu pour les personnes qui utilisent un lecteur d'écran.
+- La complexité est accrue, en donnant l'impression que l'on ne peut pas comprendre entièrement le texte sans suivre le lien pour comprendre toutes les facettes du sujet.
+
+Sortez plutôt le texte du lien du paragraphe et placez-le sur sa propre ligne pour l'aider à se démarquer et à être repéré. 
+
+**Au lieu de :**
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:46]:
```suggestion
**Au lieu de&nbsp;:** 

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+- Le flux de la phrase est interrompu pour les personnes qui utilisent un lecteur d'écran.
+- La complexité est accrue, en donnant l'impression que l'on ne peut pas comprendre entièrement le texte sans suivre le lien pour comprendre toutes les facettes du sujet.
+
+Sortez plutôt le texte du lien du paragraphe et placez-le sur sa propre ligne pour l'aider à se démarquer et à être repéré. 
+
+**Au lieu de :** 
+
+Une portion du Guide alimentaire correspond à la [quantité](https://guide-alimentaire.canada.ca/fr/) d'aliments que vous devriez consommer chaque jour dans chacun des 4 groupes alimentaires.
+
+**Écrivez :**
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:50]:
```suggestion
**Écrivez&nbsp;:**

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+Une portion du Guide alimentaire est la quantité d'aliments que vous devriez consommer chaque jour dans chacun des quatre groupes alimentaires.
+
+[Nombre de portions quotidiennes pour les enfants, les adolescents et les adultes](http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-fra.php)
+
+En retirant les liens des paragraphes et en les rendant autonomes, vous les rendez moins distrayants et plus faciles à trouver lorsque les gens effectuent le balayage de la page.
+
+### N'oubliez pas que les liens agissent comme des bretelles d'accès
+
+Les internautes consultent souvent votre page sur des appareils mobiles. Ils voient la toute petite partie de la page qui est disponible sur leur écran. Si cette page est bien structurée, ils peuvent la balayer rapidement du regard et accéder à ce dont ils ont besoin. Cependant, si vous placez des détails cruciaux après un lien, les gens risquent de ne pas s’y rendre avant de prendre la "bretelle de sortie" et de ne jamais revenir.
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:60]:
```suggestion
Les internautes consultent souvent votre page sur des appareils mobiles. Ils voient la toute petite partie de la page qui est disponible sur leur écran. Si cette page est bien structurée, ils peuvent la balayer rapidement du regard et accéder à ce dont ils ont besoin. Cependant, si vous placez des détails cruciaux après un lien, les gens risquent de ne pas s’y rendre avant de prendre la «&nbsp;bretelle de sortie&nbsp;» et de ne jamais revenir.

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+Vous pouvez «&nbsp; activer&nbsp; » ou «&nbsp; désactiver&nbsp; » les libellés de liens par le simple choix des mots. «&nbsp; Préparez votre plan de quarantaine &nbsp;» est un libellé plus attrayant que «&nbsp; Exigences provinciales et territoriales&nbsp; », car il commence par un verbe. 
+
+Lorsque vous commencez un libellé de lien par un verbe, vous rendez le texte plus actif. Lorsque vous n'utilisez que des noms, le lien semble plus passif. Utilisez cette technique pour mettre en valeur les liens les plus importants d'une page et les rendre plus attrayants que les liens qui sont là pour plus de renseignements mais qui sont moins importants pour la tâche à accomplir.
+
+Il n'est pas nécessaire que le libellé de votre lien corresponde exactement au titre de la page à laquelle il renvoie, même si les deux divent être clairement liés. Il est plus important de créer la bonne «&nbsp;piste d'information&nbsp; » pour conduire les gens là où vous voulez qu'ils aillent.
+
+Nous avons parlé de la formulation des liens dans deux de nos précédents articles de blogue :
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:86]:
```suggestion
Nous avons parlé de la formulation des liens dans deux de nos précédents articles de blogue&nbsp;: 

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+Lorsque vous commencez un libellé de lien par un verbe, vous rendez le texte plus actif. Lorsque vous n'utilisez que des noms, le lien semble plus passif. Utilisez cette technique pour mettre en valeur les liens les plus importants d'une page et les rendre plus attrayants que les liens qui sont là pour plus de renseignements mais qui sont moins importants pour la tâche à accomplir.
+
+Il n'est pas nécessaire que le libellé de votre lien corresponde exactement au titre de la page à laquelle il renvoie, même si les deux divent être clairement liés. Il est plus important de créer la bonne «&nbsp;piste d'information&nbsp; » pour conduire les gens là où vous voulez qu'ils aillent.
+
+Nous avons parlé de la formulation des liens dans deux de nos précédents articles de blogue : 
+
+- [Un exemple en faveur du texte de lien descriptif ](https://blogue.canada.ca/2020/05/26/texte-de-lien-descriptif)
+- [Étude sur les étiquettes : quels mots fonctionnent les mieux](https://blogue.canada.ca/2020/10/02/etude-sur-les-etiquettes)
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:89]:
```suggestion
- [Étude sur les étiquettes&nbsp;: quels mots fonctionnent les mieux](https://blogue.canada.ca/2020/10/02/etude-sur-les-etiquettes)

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 9]
```diff
@@ -0,0 +1,105 @@
...
+## Mot de la fin
+
+Canada.ca est la source qui permet aux Canadiens de trouver les réponses dont ils ont besoin auprès du gouvernement du Canada. Tout le monde a besoin de chemins clairs vers l'information. Éliminez les obstacles et faites en sorte qu'il soit plus facile pour les gens de trouver ces chemins grâce à une sélection minutieuse, un placement stratégique et un étiquetage clair de vos liens. 
+
+
+
+## Pour en savoir davantage
+
+- [Liens - Guide de style du contenu de Canada.ca](https://www.canada.ca/fr/secretariat-conseil-tresor/services/communications-gouvernementales/guide-redaction-contenu-canada.html#wp7-1)
+- [Recherche d'information : Une théorie sur la façon dont les gens naviguent sur le Web](https://www.nngroup.com/articles/information-foraging/) (en anglais seulement)
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:102]:
```suggestion
- [Recherche d'information&nbsp;: Une théorie sur la façon dont les gens naviguent sur le Web](https://www.nngroup.com/articles/information-foraging/) (en anglais seulement)

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 10]
```diff
@@ -0,0 +1,105 @@
...
+
+Par exemple, vous ne placeriez pas le lien vers une demande en ligne avant les détails de ce dont une personne a besoin pour remplir la demande. Il en résulterait des demandes incomplètes et une augmentation des appels de demandeurs confus qui ont suivi le lien vers la demande en ligne avant de lire les instructions. 
+
+**Réfléchissez** **bien à la séquence** que vous voulez que les gens suivent lorsque vous déterminez où placer un lien.
+
+### Faites attention aux sections Liens connexes
+
+Faites attention à ne pas enterrer au bas de la page les liens qui sont essentiels pour accomplir une tâche.
+
+Les sections «&nbsp;Liens connexes&nbsp;» au bas d'une page ont tendance à être considérées comme des informations «&nbsp;utiles à avoir&nbsp;» et non «&nbsp;indispensables&nbsp;». Dans un monde où l'information est surabondante, vous ne pouvez pas supposer que la majorité de votre public prendra le temps de parcourir ce qu'il considère seulement comme «&nbsp; utile à avoir&nbsp;».
```
[the remark this answers is not in the document]
Comment (arianna-merritt) [reply fr/_posts/2021-07-14-liens-plus-efficaces.md:70]: Thanks! I noticed those and was going to go back and fix them.

File: fr/_posts/2021-07-14-liens-plus-efficaces.md (hunk 2 above)
Comment (quidampepin) [reply fr/_posts/2021-07-14-liens-plus-efficaces.md:24]: No space before question and exclamation marks.

File: en/_posts/2021-07-14-effective-links.md [hunk 11]
```diff
@@ -0,0 +1,101 @@
...
+
+Splattering links here and there in the middle of sentences and paragraphs: 
+
+- **makes them easy to miss** when you’re scanning 
+- **interrupts the flow** of the sentence for anyone using a screen reader 
+- **adds complexity** by creating the impression that you can’t fully understand the text without following the link to get the rest of the story
+
+Instead, take the link text out of the paragraph and place it on it’s own line to help it stand out and be found. 
+
+<div class="well well-sm">
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:46]: @delisma mon div n'a pas l'air à fonctionner.

File: fr/_posts/2021-07-14-liens-plus-efficaces.md (hunk 2 above)
Comment (quidampepin) [reply fr/_posts/2021-07-14-liens-plus-efficaces.md:24]:
```suggestion
Imaginez une autoroute à 8 voies très fréquentée avec des panneaux pointant dans toutes les directions. C'est déroutant, non? Dois-je rester sur ma voie ou prendre la bretelle de sortie?! 

```

File: en/_posts/2021-07-14-effective-links.md [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+**Instead of:**
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:47]:
```suggestion
<b>Instead of:</b>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+- **adds complexity** by creating the impression that you can’t fully understand the text without following the link to get the rest of the story
+
+Instead, take the link text out of the paragraph and place it on it’s own line to help it stand out and be found. 
+
+<div class="well well-sm">
+**Instead of:** 
+
+A Food Guide serving is [how much](http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php) food you should eat from each of the 4 food groups every day.
+
+**Write:**
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:51]:
```suggestion
<b>Write:</b>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+- **makes them easy to miss** when you’re scanning 
+- **interrupts the flow** of the sentence for anyone using a screen reader 
+- **adds complexity** by creating the impression that you can’t fully understand the text without following the link to get the rest of the story
+
+Instead, take the link text out of the paragraph and place it on it’s own line to help it stand out and be found. 
+
+<div class="well well-sm">
+**Instead of:** 
+
+A Food Guide serving is [how much](http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php) food you should eat from each of the 4 food groups every day.
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:49]:
```suggestion
A Food Guide serving is <a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">how much</a> food you should eat from each of the 4 food groups every day.

```

File: en/_posts/2021-07-14-effective-links.md [hunk 15]
```diff
@@ -0,0 +1,101 @@
...
+
+**Write:**
+
+A Food Guide serving is how much food you should eat from each of the 4 food groups every day.
+
+[Number of daily food servings for children, teens and adults](http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php)
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:55]:
```suggestion
<a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">Number of daily food servings for children, teens and adults</a>

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+
+**Au lieu de :** 
+
+Une portion du Guide alimentaire correspond à la [quantité](https://guide-alimentaire.canada.ca/fr/) d'aliments que vous devriez consommer chaque jour dans chacun des 4 groupes alimentaires.
+
+**Écrivez :**
+
+Une portion du Guide alimentaire est la quantité d'aliments que vous devriez consommer chaque jour dans chacun des quatre groupes alimentaires.
+
+[Nombre de portions quotidiennes pour les enfants, les adolescents et les adultes](http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-fra.php)
```
Comment (quidampepin) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:55]: Put this in a well (see English) - everything within it has to be HTML.

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+
+Inclure beaucoup de de bretelles d'accès différentes est un moyen sûr d'égarer vos utilisateurs dans votre contenu Web.
+
+**Réduisez la fatigue liée aux décisions**. N'ajoutez que les liens qui aideront réellement les gens à accomplir leur tâche. 
+
+Si le lien ne soutient pas vraiment la tâche de la page, ne le mettez pas sur la page.
+
+## Placez les liens de façon à ce que les gens les trouvent
+
+L'emplacement des liens sur une page a un impact important sur leur efficacité.
```
Comment (quidampepin) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:34]:
```suggestion
L'emplacement des liens sur une page a des répercussions importantes sur leur efficacité.

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+## Rédigez de bons libellés de liens
+
+Vos libellés de liens doivent être descriptifs. Ils doivent être autonomes et indiquer clairement ce que les utilisateurs peuvent s'attendre à trouver s'ils suivent le lien. 
+
+Vous pouvez «&nbsp;activer&nbsp; » ou «&nbsp;désactiver&nbsp;» les libellés de liens par le simple choix des mots. «&nbsp;Préparez votre plan de quarantaine&nbsp;» est un libellé plus attrayant que «&nbsp;Exigences provinciales et territoriales&nbsp;», car il commence par un verbe. 
+
+
+Lorsque vous commencez un libellé de lien par un verbe, vous rendez le texte plus actif. Lorsque vous n'utilisez que des noms, le lien semble plus passif. Utilisez cette technique pour mettre en valeur les liens les plus importants d'une page et les rendre plus attrayants que les liens qui sont là pour plus de renseignements mais qui sont moins importants pour la tâche à accomplir.
+
+Il n'est pas nécessaire que le libellé de votre lien corresponde exactement au titre de la page à laquelle il renvoie, même si les deux divent être clairement liés. Il est plus important de créer la bonne «&nbsp;piste d'information&nbsp; » pour conduire les gens là où vous voulez qu'ils aillent.
```
Comment (quidampepin) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:88]:
```suggestion
Il n'est pas nécessaire que le libellé de votre lien corresponde exactement au titre de la page à laquelle il renvoie, même si les deux doivent être clairement liés. Il est plus important de créer la bonne «&nbsp;piste d'information&nbsp; » pour conduire les gens là où vous voulez qu'ils aillent.

```

**Review by quidampepin — CHANGES_REQUESTED** [review]

File: en/_posts/2021-07-14-effective-links.md [hunk 19]
```diff
@@ -0,0 +1,104 @@
...
+Splattering links here and there in the middle of sentences and paragraphs: 
+
+- **makes them easy to miss** when you’re scanning 
+- **interrupts the flow** of the sentence for anyone using a screen reader 
+- **adds complexity** by creating the impression that you can’t fully understand the text without following the link to get the rest of the story
+
+Instead, take the link text out of the paragraph and place it on it’s own line to help it stand out and be found. 
+
+<div class="well well-sm">
+<b>Instead of:</b>
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:47]:
```suggestion
<p><b>Instead of:</b></p>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 20]
```diff
@@ -0,0 +1,104 @@
...
+
+
+A Food Guide serving is <a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">how much</a> food you should eat from each of the 4 food groups every day.
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:50]:
```suggestion
<p>A Food Guide serving is <a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">how much</a> food you should eat from each of the 4 food groups every day.</p>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 21]
```diff
@@ -0,0 +1,104 @@
...
+
+
+<b>Write:</b>
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:53]:
```suggestion
<p><b>Write:</b></p>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 22]
```diff
@@ -0,0 +1,104 @@
...
+
+
+A Food Guide serving is how much food you should eat from each of the 4 food groups every day.
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:56]:
```suggestion
<p>A Food Guide serving is how much food you should eat from each of the 4 food groups every day.</p>

```

File: en/_posts/2021-07-14-effective-links.md [hunk 23]
```diff
@@ -0,0 +1,104 @@
...
+
+<a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">Number of daily food servings for children, teens and adults</a>
```
Comment (quidampepin) [inline en/_posts/2021-07-14-effective-links.md:58]:
```suggestion
<p><a href="http://www.hc-sc.gc.ca/fn-an/food-guide-aliment/basics-base/quantit-eng.php">Number of daily food servings for children, teens and adults</a></p>

```

**Review by quidampepin — CHANGES_REQUESTED** [review]
Adding paragraph tags in the well

File: en/_posts/2021-07-14-effective-links.md [hunk 24]
```diff
@@ -0,0 +1,108 @@
+---
+altLangPage: "/2021/07/14/liens-plus-efficaces"
+date: 2021-07-14
+description: " "
```
Comment (delisma) [inline en/_posts/2021-07-14-effective-links.md:4]: Description is missing

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 25]
```diff
@@ -0,0 +1,113 @@
...
+
+
+- Les liens sont faciles à manquer lors du balayage de la page. 
+- Le flux de la phrase est interrompu pour les personnes qui utilisent un lecteur d'écran.
+- La complexité est accrue, en donnant l'impression que l'on ne peut pas comprendre entièrement le texte sans suivre le lien pour comprendre toutes les facettes du sujet.
+
+Sortez plutôt le texte du lien du paragraphe et placez-le sur sa propre ligne pour l'aider à se démarquer et à être repéré. 
+
+<div class="well well-sm">
+<p><b>Au lieu de :</b></p>
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:50]:
```suggestion
<p><b>Au lieu de&nbsp;:</b></p> 

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 26]
```diff
@@ -0,0 +1,113 @@
...
+- La complexité est accrue, en donnant l'impression que l'on ne peut pas comprendre entièrement le texte sans suivre le lien pour comprendre toutes les facettes du sujet.
+
+Sortez plutôt le texte du lien du paragraphe et placez-le sur sa propre ligne pour l'aider à se démarquer et à être repéré. 
+
+<div class="well well-sm">
+<p><b>Au lieu de :</b></p> 
+
+<p>Une portion du Guide alimentaire correspond à la <a href="https://guide-alimentaire.canada.ca/fr/">quantité</a> d'aliments que vous devriez consommer chaque jour dans chacun des 4 groupes alimentaires.</p>
+
+<p><b>Écrivez : </b></p>
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:54]:
```suggestion
<p><b>Écrivez&nbsp;:</b></p>

```

**Review by delisma — CHANGES_REQUESTED** [review]

File: en/_posts/2021-07-14-effective-links.md [hunk 27]
```diff
@@ -0,0 +1,100 @@
...
+
+### Be careful with Related links sections
+
+Be careful about burying links that are crucial to completing a task at the bottom of the page.
+
+“Related Links” sections at the bottom of a page tend to be seen as “nice-to-have” not “need-to-have” information. In a world of information overload, you can’t assume that the majority of your audience will take the time to peruse anything they consider only “nice-to-have.” 
+
+On the web, people are on a mission and they’re moving fast. If you’re providing a link that is crucial to their task, make that obvious by placing it within the flow of the task, not in a “Related links” section.
+
+> “Related links are random links that are there” - usability research participant
```
Comment (delisma) [inline en/_posts/2021-07-14-effective-links.md:76]:
```suggestion
> “Related links are random links that are there” 
> - usability research participant

```

File: fr/_posts/2021-07-14-liens-plus-efficaces.md [hunk 28]
```diff
@@ -0,0 +1,100 @@
...
+
+### Faites attention aux sections Liens connexes
+
+Faites attention à ne pas enterrer au bas de la page les liens qui sont essentiels pour accomplir une tâche.
+
+Les sections «&nbsp;Liens connexes&nbsp;» au bas d'une page ont tendance à être considérées comme des informations «&nbsp;utiles à avoir&nbsp;» et non «&nbsp;indispensables&nbsp;». Dans un monde où l'information est surabondante, vous ne pouvez pas supposer que la majorité de votre public prendra le temps de parcourir ce qu'il considère seulement comme «&nbsp;utile à avoir&nbsp;». 
+
+Sur le Web, les gens sont en mission et se déplacent rapidement. Si vous fournissez un lien qui est crucial pour leur tâche, rendez-le évident en le plaçant dans le flux de la tâche, et non dans une section «&nbsp;Liens connexes&nbsp;».
+
+> «&nbsp;Les liens connexes sont des liens aléatoires qui se trouvent simplement là&nbsp;» - participant à une recherche sur la convivialité
```
Comment (delisma) [inline fr/_posts/2021-07-14-liens-plus-efficaces.md:76]:
```suggestion
> «&nbsp;Les liens connexes sont des liens aléatoires qui se trouvent simplement là&nbsp;» 
> - participant à une recherche sur la convivialité

```

**Review by delisma — APPROVED** [review]

**Review by delisma — APPROVED** [review]

**Pull request merged**